src/components/custom-field-input-text/custom-field-input-text.jsx

```diff
@@ -39,7 +39,7 @@ export default function CustomFieldInputText(props) {
   });
 
   return (
-    <div className={getRootClassName(props.className, props.error, props.disabled)}>
+    <div className={getRootClassName(props.className, props.error, props.disabled)} data-testid="custom-field-input" >
```

Comment:
Should the `testid` here by dynamic to support testing of multiple input elements? Maybe combine with `id`, or some auto-generated UUID?

Reply:
I am not sure. I didn't have a use-case for that. I am not sure if tests with multiple custom field inputs should be using IDs to assert on the containers. I think instead those tests should be asserting on the state of their forms.